Formula/p/proteinortho.rb

```diff
@@ -16,19 +16,34 @@ class Proteinortho < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "bde04add25030aed32aac06465de994ca04e4410181a53deac825858df3df18c"
   end
 
+  depends_on "libomp" => :build
   depends_on "diamond"
-  depends_on "libomp"
   depends_on "openblas"
+  on_linux do
+    depends_on "gcc" # gcc is not allowed on mac
+  end
```

Comment:
although it should be ordered alphabetically, this is the result of `brew style Formula/p/proteinortho.rb --fix`